test/build.gradle.kts

```diff
@@ -54,6 +54,10 @@ kotlin {
 
         getByName("commonTest") {
             dependencies {
+                // TODO AtomicFu doesn't work on the test project. It throws:
+                //  Source set 'androidMain' of project 'test' is part of several compilations [debug, release]
```

Comment:
https://github.com/Kotlin/kotlinx.atomicfu/issues/90#issuecomment-597872907 :(

Reply:
Bummer. I'll add a reference to it in the comment